Priorities/Views/AddPlayerPage.xaml

```diff
@@ -0,0 +1,136 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="Priorities.Views.AddPlayerPage"
+             Title="Add Player"
+             Shell.FlyoutBehavior="Flyout">
+    
+    
+    <ScrollView>
+        <Grid Padding="0" RowSpacing="20" BackgroundColor="White">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="auto" />
+                <RowDefinition Height="auto" />
+                <RowDefinition Height="auto" />
+            </Grid.RowDefinitions>
+
+            <!--Header-->
+            <!--<Grid Grid.Row="1" ColumnSpacing="3" RowSpacing="10">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="auto" />
+                    <ColumnDefinition Width="auto" />
+                    <ColumnDefinition Width="auto" />
+                </Grid.ColumnDefinitions>
+
+                <Button Grid.Row="0"
+                        Grid.Column="0"
+                        Text="Hamburger" />
+
+                <Label Grid.Row="0"
+                       Grid.Column="1"
+                       Text="Add Player" />
+
+                <Button Grid.Row="0"
+                        Grid.Column="2"
+                        Text="X" />
+            </Grid>-->
+
+            <!--Main Section-->
+
+            <!--Name Section-->
+            <Grid Grid.Row="0" Padding="12">
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="auto" />
+                    <RowDefinition Height="auto" />
+                </Grid.RowDefinitions>
+
+                <!--Label Name-->
+                <Label Grid.Row="0"
+                       Text="Name"
+                       HorizontalOptions="Center"
+                       FontSize="28"
+                       TextColor="Black"
```

Comment:
On dark mode devices, the default text color is white so you can't see it on the white background.